opds_import.py

```diff
@@ -844,7 +848,8 @@ def consolidate_links(cls, links):
 
         Similarly if link n is a thumbnail and link n+1 is an image.
         """
-        new_links = list(links)
+        new_links = [x for x in links if x]
+        links = list(new_links)
```

Comment:
It's not that I'm converting it into a list, it's that I'm making another copy of the list. I need `new_links` to be a copy of `links`, because I'm going to go through `links`, consolidate thumbnails, and remove the thumbnails from `new_links`. I can't iterate over `new_links` because I'm going to be changing the list.

Do you have any ideas about how to convey this better?